src/components/thumbnail/react/Thumbnail.tsx

```diff
@@ -50,6 +71,8 @@ interface IThumbnailProps extends IGenericProps {
     image: string;
     /* Size. */
     size?: ThumbnailSize;
+    /* Image Loading. */
+    loading?: ImageLoading;
```

Comment:
Could you transform all the props comments to JSDoc comments (`/*` => `/**`)?
Otherwise they cannot be detected in the prop table I think.

Reply:
@gcornut done!

<img width="915" alt="Screenshot 2019-10-11 at 17 44 51" src="https://user-images.githubusercontent.com/13719066/66665498-f7c50800-ec4e-11e9-94bc-d69a58afed9c.png">